docker/docker_image_src.go

```diff
@@ -474,24 +474,22 @@ func deleteImage(ctx context.Context, sys *types.SystemContext, ref dockerRefere
 		return errors.Errorf("Failed to delete %v: %s (%v)", deletePath, string(body), delete.Status)
 	}
 
-	if c.signatureBase != nil {
-		manifestDigest, err := manifest.Digest(manifestBody)
+	manifestDigest, err := manifest.Digest(manifestBody)
```

Comment:
OK; please document for `newDockerClientFromRef` that in the return value, `signatureBase` is always set.